### PR TITLE
Update awsses service And send grid

### PR DESCRIPTION
**Pull request by agrawalprakhar**

### CODE REVIEW

File: EmailService.SES/SESEmailService.cs [hunk 1]
```diff
@@ -23,52 +32,193 @@ public SESEmailService(IOptions<SESOptions> sesOptions)
...
-                        Data = mail.Body
-                    };
+                        builder.Attachments.Add(attachment.FileName, attachment.Content, ContentType.Parse(attachment.ContentType));
+                    }
                 }
 
-                else
+                message.Body = builder.ToMessageBody();
+
+                var stream = new MemoryStream();
```
Comment (iroshni) [inline EmailService.SES/SESEmailService.cs]: Please use using keyword for disposing of the object.

File: EmailService/Mail.cs [hunk 2]
```diff
@@ -1,21 +1,86 @@
 ﻿using System.Collections.Generic;
+using System.Net.Mail;
 
 namespace EmailService
 {
-    public class Mail
+    public class EMail
```
Comment (iroshni) [inline EmailService/Mail.cs]: Rename to EmailBase

File: EmailService/Mail.cs [hunk 3]
```diff
@@ -1,21 +1,86 @@
...
+        public List<EmailAddress> CC { get; set; }
+        public List<EmailAddress> BCC { get; set; }
+
+        public List<AttachmentData>? Attachments { get; set; } = new List<AttachmentData>();
+
+        public bool IsBodyHTML { get; set; }
+
+    }
+
+    public class Mail : EMail
```
Comment (iroshni) [inline EmailService/Mail.cs]: Rename to Email and inherit EmailBase

File: EmailService/Mail.cs [hunk 4]
```diff
@@ -1,21 +1,86 @@
...
+            BCC = bcc ?? new List<EmailAddress>();
+            Attachments = attachments ?? new List<AttachmentData>();
+        }
+
+        public string Body { get; set; }
 
         public string Subject { get; set; }
+    }
+
+    public class TemplatedEmailRequest : EMail
```
Comment (iroshni) [inline EmailService/Mail.cs]: Inherit Email Base

File: EmailService/Mail.cs (hunk 2 above)
Comment (agrawalprakhar) [reply EmailService/Mail.cs]: Fixed

**Review by iroshni — APPROVED** [review]

**Pull request merged**